### DISCUSSION TITLE
extended desktop

### DISCUSSION THREAD

**Discussion by alverman** [sumatrapdfreader/sumatrapdf#3480] [Q&A]
good morning,
having the pc in extended mode would it be possible to display the pdf in both monitors ?

Thank you, Alberto

**Comment by GitHubRulesOK** [comment]
Yes, but depends what you need.
The current pre-release has tear off tabs like a browser, so that is one method. Another is two sessions, so one on each monitor. That then has maintenance issues as two separate settings and histories, but one could be a temporary 2nd new window from the first (ctrl shift N), there are other ways but those should give enough choice.
![image](https://user-images.githubusercontent.com/12754913/236296944-7af32381-df2d-4c9a-a0f1-0eeb1cf0f965.png)

**Reply by alverman** [reply]
I control the speaker's pc from a program I am developing, so I run the ppts, pdfs, and videos remotely.
The problem occurs to me when I run the pdfs, which I only display on the extended monitor, but at that point the speaker does not see it on the pc monitor.
I have tried adding a function that clones the displays but what I can't get rid of is the windows side band when I clone the screens to see the pdf on the two screens.

So I was asking if there was a way to view the pdf on both screens when extended.

**Reply by alverman** [reply]
Sumatra I open it with the -presentation switch.
I thought that this way it would start in double

**Reply by GitHubRulesOK** [reply]
Presentation is one full window for projection screening, so not what you need but will help.

Are you needing one page shows on two screens with a command to go forward or back a number of pages together ?

then that is where two copies with one as controller should help.
Perhaps take a look at end of this discussion THEN if usefull  jump back to halfway up then move down with tests
https://forum.sumatrapdfreader.org/t/current-page-number-of-the-page-displayed-for-presentations/4144

**Reply by alverman** [reply]
..... check
